Python Match Simulator/RobotQuality.py

```diff
@@ -45,6 +45,52 @@
             "Reliability": 92
         }
     },
+    "GOOD": {

```

Comment:
Generally just tried to pick cycle times and reliabilities between BEST and AVERAGE